docs/azure-setup.md

```diff
@@ -79,7 +79,7 @@ An ARM template can be opened within Azure Portal via the following REST API:
 
 The URI can be one for either an Azure Blob or a GitHub file.  For example,
 
-    https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Ftongtest.blob.core.windows.net%2Fcntk%2FazureDeployMainTemplate.json
+    https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fmmlspark%2Fmaster%2Ftools%2Fdeployment%2FazureDeployMainTemplate.json
```

Comment:
Talked more about the two issues mentioned here with Eli:
- GitHub urls used in the doc example  and in the master deployment template 
- Url to a particular version of script action hardcoded in the deployment templates.

The current plan is to have the build process generate the final version of the templates using the checked-in version as the template where the build process will fill in the latest version of the script action and references to the child deployment templates. Since the templates are hosted at  https://mmlspark.azureedge.net, internal and external reference will not involve any GitHub urls and the urls to the script actions are updated.

I will undo the change here.